droopler.install

```diff
@@ -1681,3 +1681,53 @@ function droopler_update_8124() {
   $updater->checklist()
     ->saveProgress(['v2.2' => $values]);
 }
+
+/**
+ * Droopler 2.2 - remove "mt-5" spacing from CTA field in Reference Content Paragraph.
+ */
+function droopler_update_8125() {
+  /** @var Drupal\d_update\Updater $updater */
+  $updater = \Drupal::service('d_update');
+
+  $values['droopler__droopler_update_8125_1'] = update_modules_configuration(['d_p_reference_content'], '8125');
+  $updater->checklist()->saveProgress(['v2.2' => $values]);
+}
+
+/**
+ * Droopler 2.2 - remove "mt-5" spacing from CTA field saved in database.
+ */
+function droopler_update_8126(&$sandbox) {
+  /** @var Drupal\d_update\Updater $updater */
+  $updater = \Drupal::service('d_update');
+  $limit = 2;
+  if (!isset($sandbox['current'])) {
+    $sandbox['current'] = 0;
+  }
+  $entities = \Drupal::entityQuery('paragraph')
+    ->condition('type', 'd_p_reference_content')
+    ->range($sandbox['current'], $limit)
+    ->execute();
+  $sandbox['total'] = count($entities);
```

Comment:
moment, co tu się w ogóle podziało :scream: 
czy to teraz nie zadziała przypadkiem tak, że ten kod zaktualizuje tylko 2 paragrafy ?
to `total` powinno byc ustawione za pierwszym razem, w tym `if` na poczatku - bez dodawania zadnego range..
range dopiero powinno byc dodawane przy pobieraniu idikow..

Reply:
A dlaczego total powinno być przed if skoro skoro jest wykorzystywane do sprawdzenia czy należy pobrać kolejna porcję danych do przetworzenia ? total nie ma nic wspólnego z range ?

Reply:
no ale popatrz na cały przebieg funkcji:
![image](https://user-images.githubusercontent.com/37533838/93064836-50c7c800-f678-11ea-8bad-5b737f414867.png)